sbnana/CAFAna/Core/SpectrumLoaderBase.h

```diff
@@ -257,6 +311,10 @@ namespace ana
 
     typedef _VarOrMultiVar<caf::SRSliceProxy> VarOrMultiVar;
     typedef _VarOrMultiVar<caf::SRSpillProxy> SpillVarOrMultiVar;
+    typedef _VarOrMultiVar<caf::SRTrueInteractionProxy> TruthVarOrMultiVar;
+
+    template<class T>
+    friend bool operator<(const _VarOrMultiVar<T>& a, const _VarOrMultiVar<T>& b) noexcept;
```

Comment:
Isn't this just a repeat of Lines 319-320?

Reply:
Thanks, this is removed.